src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,14 @@
+export default function ArticleCard({ article }) {
+  return (
+    <li className="article-card">
+      <h3 className="article-card-title">{article.title}</h3>
+      <img className="article-card-image" src={article.article_img_url} />
+      <div className="article-card-footer">
+        <p className="article-card-topic">Topic: {article.topic}</p>
+        <p className="article-card-author">Author: {article.author}</p>
+      </div>
+    </li>
+  );
+}
+
```

Comment:
Nice! Consider adding comment counts, likes and the date published. You could also use icons to label these if you'd like. "💬 7" for example.